java/integTest/src/test/java/glide/TestUtilities.java

```diff
@@ -3,22 +3,32 @@
 
 import glide.api.models.BaseTransaction;
 import glide.api.models.commands.SetOptions;
+import java.util.Map;
 import java.util.UUID;
 
 public class TestUtilities {
+    private static final String key1 = "{key}" + UUID.randomUUID();
+    private static final String key2 = "{key}" + UUID.randomUUID();
+    private static final String key3 = "{key}" + UUID.randomUUID();
```

Comment:
is there a reason key3 is being created but not used?

Reply:
I was copying over the transaction tests from Node and Py must have copied one extra I'll remove it.